Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ron"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
Would you like to make a release? I am hitting this exact issue right now :rofl: